config/initializers/omniauth.rb

```diff
@@ -18,7 +18,7 @@
   if AppConfig.services.facebook.enable?
     provider :facebook, AppConfig.services.facebook.app_id, AppConfig.services.facebook.secret, {
       display: 'popup',
-      scope: 'publish_actions,publish_stream,offline_access',
+      scope: 'public_profile,publish_actions',
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.